integrated_channels/cornerstone/exporters/content_metadata.py

```diff
@@ -50,3 +51,12 @@ def transform_description(self, content_metadata_item):
         if 0 < len(full_description) <= self.LONG_STRING_LIMIT:  # pylint: disable=len-as-condition
             return full_description
         return content_metadata_item.get('short_description') or content_metadata_item.get('title') or ''
+
+    def transform_subjects(self, content_metadata_item):
+        """
+        Return the transformed version of the course subject list.
+        """
+        subjects = []
+        for subject in content_metadata_item.get('subjects', []):
+            subjects.append(CORNERSTONE_SUBJECTS_MAPPING.get(subject, self.DEFAULT_SUBJECT))
```

Comment:
nit: we probably perform case in-sensitive comparison to match `Design` and `design`.